test/workbox-routing/node/environment.mjs

```diff
@@ -1,29 +1,18 @@
-import clearRequire from 'clear-require';
-import makeServiceWorkerEnv from 'service-worker-mock';
 import sinon from 'sinon';
 import {expect} from 'chai';
-
+import clearRequire from 'clear-require';
 import expectError from '../../../infra/utils/expectError';
 
 describe(`workbox-routing: SW environment`, function() {
   const sandbox = sinon.sandbox.create();
-  const initialNodeEnv = process.env.NODE_ENV;
-
-  before(function() {
-    Object.assign(global, makeServiceWorkerEnv());
-  });
-
-  beforeEach(function() {
-    clearRequire.all();
-  });
 
   afterEach(function() {
     sandbox.restore();
-    process.env.NODE_ENV = initialNodeEnv;
+    clearRequire.all();
```

Comment:
Is this still needed? Normally I was using this to handle changes in dev / prod.

Reply:
I think it is? I'm assuming the throw behavior will only happen the first time the module is evaluated, so if we have multiple tests checking for a throw, we'll need to clear the require cache (or whatever that's doing).